testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java

```diff
@@ -105,6 +105,7 @@ public static void test(String root, Set<String> unreferenced, Set<String> unuse
                 invalidUnref.add(unref);
             }
         }
+        // If error an occurs please have a look at "unreferenced" java doc
         if (!invalidUnref.isEmpty()) {
             throw new Exception("Some unreferenced modules are un-expected " + invalidUnref);
```

Comment:
This should still provide some context, e.g. the path to the 'test-standalone-reference' and 'test-all-layers' installations that were compared, e.g.

"Some unreferenced modules in installation <path to test-all-layers> are unexpected compared to <path to test-standalone-reference> -- "

Reply:
@parsharma I meant in the exception message.

Reply:
@parsharma I meant in the exception message.